### PR TITLE
Add review form

### PR DESCRIPTION
**Pull request by chaines**
Basically done. There's some jank for sure.

Don't have the picture upload functionality

### CODE REVIEW

File: src/components/reviews/AddReviewModal.js [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+  };
+
+  const setCharacteristic = (index, value) => {
+    const newChars = [...characteristics];
+    newChars[index] = value;
+    setCharacteristics(newChars);
+  };
+
+  const validateBody = () => {
+    console.log('brrr');
```
Comment (Katerina-spb) [inline src/components/reviews/AddReviewModal.js]: 😂

File: src/components/reviews/AddReviewModal.js [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+      product_id: props.productId,
+      rating,
+      summary,
+      body,
+      recommend: recommend === 'true',
+      name: nickname,
+      email,
+      characteristics: charObj,
+    };
+    console.log(JSON.stringify(params, null, 2));
```
Comment (Katerina-spb) [inline src/components/reviews/AddReviewModal.js]: 🧐

File: src/components/reviews/AddReviewModal.js [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+    <Modal open={props.open} onClose={handleClose} className={classes.addReviewModal}>
+      <div className={classes.modalContent}>
+        <h1>Add a Review</h1>
+        <form onSubmit={handleSubmit}>
+          <FormControl component="fieldset">
+            <Rating
+              name="new-review-rating"
+              value={rating}
+              onChange={(e, v) => {
+                console.log(v);
```
Comment (Katerina-spb) [inline src/components/reviews/AddReviewModal.js]: 👻

File: src/components/reviews/ReviewsList.js [hunk 4]
```diff
@@ -1,42 +1,61 @@
...
+
+  const addReview = (event) => {
+    setModalOpen(true);
+  };
+  const closeModal = (event) => {
+    setModalOpen(false);
+  };
   return (
     <div className={`${props.className}`}>
+      {console.log(props.data)}
```
Comment (Katerina-spb) [inline src/components/reviews/ReviewsList.js]: 👀

**Review by Katerina-spb — APPROVED** [review]

**Review by Hkanabar — COMMENTED** [review]
I don't think I noticed anything glaring that could cause issues. Should be good to merge.

File: src/components/reviews/AddReviewModal.js [hunk 5]
```diff
@@ -0,0 +1,216 @@
...
+      summary,
+      body,
+      recommend: recommend === 'true',
+      name: nickname,
+      email,
+      characteristics: charObj,
+    };
+    console.log(JSON.stringify(params, null, 2));
+    apiController.postReview(params).then((res) => {
+      console.log(res);
```
Comment (chaines) [inline src/components/reviews/AddReviewModal.js]: You missed one @Katerina-spb

**Pull request merged**